internal/backends/winit/winitwindowadapter.rs

```diff
@@ -1134,13 +1190,19 @@ impl WindowAdapterInternal for WinitWindowAdapter {
 
     #[cfg(muda)]
     fn setup_menubar(&self, menubar: vtable::VBox<i_slint_core::menus::MenuVTable>) {
-        drop(self.muda_adapter.borrow_mut().take());
-        self.muda_adapter.replace(Some(crate::muda::MudaAdapter::setup(
-            menubar,
-            &self.winit_window().unwrap(),
-            self.event_loop_proxy.clone(),
-            self.self_weak.clone(),
-        )));
+        drop(self.menubar.borrow_mut().take());
+        self.menubar.replace(Some(menubar));
```

Comment:
The drop-then-replace dance was done because on Windows, we deeded to drop the muda menu before re-creating a new one.
Here, the setup is no longer called so we can just do

```suggestion
        self.menubar.replace(Some(menubar));
```